packages/in_app_purchase/example/README.md

```diff
@@ -85,4 +85,4 @@ below.
    service (including iTunes!) with the test account will permanently invalidate
    it. Sign in to the test account in the example app following the steps in the
    [*In-App Purchase Programming
-   Guide*](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/StoreKitGuide/Chapters/ShowUI.html#//apple_ref/doc/uid/TP40008267-CH3-SW11).
\ No newline at end of file
+   Guide*](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/StoreKitGuide/Chapters/ShowUI.html#//apple_ref/doc/uid/TP40008267-CH3-SW11).
```

Comment:
What's diff here? I couldn't find any diffs :)

Reply:
Looks like trailing whitespace on the old version.